code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,293 @@
+					new /obj/item/twohanded/fireaxe(src)
+					new /obj/item/organ/brain/alien(src)
+				if("heavystuff4")
+					new /obj/item/modular_computer/tablet/preset/advanced(src)
+				if("heavystuff5")
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/clothing/head/collectable/tophat(src)
+					new /obj/item/storage/fancy/cigarettes/cigars/cohiba(src)
+				if("heavystuff6")
+					new /obj/item/borg/upgrade/syndicate(src)
+				if("heavystuff7")
+					new /obj/item/borg/upgrade/selfrepair(src)
+				if("heavystuff8")
+					new /obj/item/clothing/suit/armor/riot/knight/red(src)
+					new /obj/item/clothing/gloves/plate/red(src)
+					new /obj/item/clothing/shoes/plate/red(src)
+					new /obj/item/clothing/head/helmet/knight/red(src)
+					new /obj/item/kitchen/knife/butcher(src)
+		if(114)
+			for(var/food in 5)
+				food = pick(subtypesof(/obj/item/reagent_containers/food/snacks) - /obj/item/reagent_containers/food/snacks)
+				new food(src)
+		if(115)
+			new /obj/item/kitchen/knife/rainbowknife(src)//Yooo my 400000$ knife lets sell it and become rich!!!!11
+		if(116)
+			switch(pickweight(list("litestuff" = 3, "litestuff2" = 2, "litestuff3" = 1, "litestuff4" = 4, "litestuff5" = 1)))
+				if("litestuff")
+					new /obj/item/melee/classic_baton/police(src)
+					new /obj/item/storage/firstaid/regular(src)
+					new /obj/item/storage/toolbox/syndicate(src)
+				if("litestuff2")
+					new /obj/item/melee/classic_baton/police/telescopic(src)
+					new /obj/item/storage/firstaid/toxin(src)
+					new /obj/item/grenade/clusterbuster/smoke(src)
+					new /obj/item/clothing/shoes/chameleon/noslip(src)
+				if("litestuff3")
+					new /obj/item/clothing/under/chameleon(src)
+					new /obj/item/clothing/neck/necklace/dope(src)
+					new /obj/item/clothing/mask/chameleon(src)
+				if("litestuff4")
+					new /obj/item/storage/firstaid/regular(src)
+					new /obj/item/storage/firstaid/toxin(src)
+					new /obj/item/storage/firstaid/brute(src)
+					new /obj/item/storage/firstaid/fire(src)
+					new /obj/item/borg/upgrade/ddrill(src)
+					new /obj/item/borg/upgrade/soh(src)
+				if("litestuff5")
+					new /obj/item/grenade/clusterbuster/smoke(src)
+		if(117 to 120)
+			new /obj/item/tank/jetpack/void(src)
+			new /obj/item/clothing/mask/breath(src)
+			new /obj/item/clothing/head/helmet/space/nasavoid(src)
+			new /obj/item/clothing/suit/space/nasavoid(src)
+			new /obj/item/crowbar(src)
+			new /obj/item/stock_parts/cell(src)
+			new /obj/item/multitool(src)
+		if(121)
+			new /obj/item/clothing/gloves/chameleon/combat(src)
+			for(var/gloves in 6)
+				gloves = pick(subtypesof(/obj/item/clothing/gloves/color) - /obj/item/clothing/gloves/color)
+				new gloves(src)
+		if(122)
+			new /obj/item/clothing/gloves/gang(src)
+		if(123)
+			for(var/pizza in 5)
+				pizza = pick(subtypesof(/obj/item/pizzabox))
+				new pizza(src)
+		if(124)
+			new /obj/item/clothing/shoes/chameleon/noslip(src)
```

Comment:
Antagonist items should not be in these crates considering they are easy to open and encourage meta programs